cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -109,7 +110,21 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 			return false
 		}
 		defer resp.Body.Close()
-		return resp.StatusCode == http.StatusOK
```

Comment:
may want to keep this as early exit condition `if resp.StatusCode != http.StatusOK`